config.yaml

```diff
@@ -0,0 +1,11 @@
+stats:
+  interval: 5 #second
+  api-url: ""
+jsonrpc:
+  miner:
+    url: "http://167.172.98.186:2345/rpc/v0"
```

Comment:
This will be localhost 127.0.0.1 for every user. However, can you write config on the fly or have multiple configs for local development and so?

Reply:
That is the idea, I am still investigating the best approach to have this implemented.